reflectiz/manifest.json

```
@@ -0,0 +1,63 @@
+{
+  "manifest_version": "2.0.0",
+  "display_on_public_website": false,
+  "app_id": "reflectiz",
+  "app_uuid": "79767e7d-f5db-4528-aeea-1cd68649ccd8",
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "uninstallation": "README.md#Uninstallation",
+    "changelog": "CHANGELOG.md",
+    "description": "The Reflectiz integration provides security insights for your website.",
+    "title": "Reflectiz",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Web Exposure Alerts Dashboard",
+        "image_url": "images/dashboard_1.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Web Exposure Rating Dashboard",
+        "image_url": "images/dashboard_2.png"
+      }
```

Swap order to display image with logo first. 

```suggestion
      {
        "media_type": "image",
        "caption": "Web Exposure Rating Dashboard",
        "image_url": "images/dashboard_2.png"
      },
      {
        "media_type": "image",
        "caption": "Web Exposure Alerts Dashboard",
        "image_url": "images/dashboard_1.png"
      }
```